app/controllers/green_lanes/applicable_exemptions_controller.rb

```diff
@@ -8,7 +8,9 @@ class ApplicableExemptionsController < ApplicationController
 
     def new
       @exemptions_form = build_exemptions_form
-      @back_link_path = determine_back_link_path
+      @category_one_assessments_without_exemptions = candidate_categories.cat1_without_exemptions
+      @category_two_assessments_without_exemptions = candidate_categories.cat2_without_exemptions
```

Comment:
are the
@category_one_assessments_without_exemptions 
@category_two_assessments_without_exemptions 
used in the UI (Erb files)
?
maybe we do not need them ( BackLinkPath can get those directly or via args)

Reply:
yeah i guess so